PayPal.xcodeproj/xcshareddata/xcschemes/PayPalUITests.xcscheme

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1330"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
+      buildImplicitDependencies = "YES">
+   </BuildAction>
+   <TestAction
+      buildConfiguration = "Debug"
+      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
+      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      shouldUseLaunchSchemeArgsEnv = "YES">
+      <Testables>
+         <TestableReference
+            skipped = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "BCFAC70A27ED043100C3AF00"
+               BuildableName = "PayPalTests.xctest"
```

Comment:
`PayPalTests` > `PayPalUITests`?

Reply:
Yeah this is odd. Can I change this value from an Xcode GUI?

Reply:
Hmmm. Looks like this once didn't get updates. May need to just update it manually outside of the GUI?

Reply:
It should be good now with the caveat of Xcode adding more xml to the file. I guess it's necessary, might have to do with the new version of Xcode.